db/migrate/20211118032523_create_orders.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOrders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.references :user, null: false, foreign_key: true
+      t.integer :quantity
+      t.boolean :status
```

Comment:
order chỉ có 2 trạng thái thôi à e?

Reply:
dạ order có trạng thái ở status là  đặt hoặc chưa đặt , và is_payment: thanh toán hoặc chưa thanh toán ạ